ansible/group_vars/nimbus.holesky.yml

```
@@ -153,13 +153,27 @@ nimbus_era_files_network: '{{ beacon_node_network }}'
 nimbus_era_files_db_path: '/data/beacon-node-{{ beacon_node_network }}-unstable/data/db'
 nimbus_era_files_nclidb_path: '/data/beacon-node-{{ beacon_node_network }}-unstable/repo/build/ncli_db'
 
+# Migrated to NFTables from IPTables.
+# https://github.com/status-im/infra-misc/issues/301
+bootstrap__firewall_nftables: true
+
 # Open Ports -------------------------------------------------------------------
-open_ports_default_chain: 'VPN'
+host_el_type: '{{ ansible_hostname|split("-")|first }}'
 open_ports_list:
-  - { port: '80',          chain: 'SERVICES',    comment: 'Nginx' }
-  - { port: '443',         chain: 'SERVICES',    comment: 'Nginx' }
-  - { port: '6060:6070',   ipset: 'metrics.hq',  comment: 'EL Metrics' }
-  - { port: '9400:9404',   ipset: 'metrics.hq',  comment: 'Geth Exporter' }
-  - { port: '9300:9310',   ipset: '{{ env }}.{{ stage }}', comment: 'Nimbus REST API' }
+  nginx:
+    - { port: 443, comment: 'Nginx' }
+  exec-node:
+    - { port: '30303-30315', comment: '{{ host_el_type }} libp2p stable',    protocol: 'tcp'                     }
+    - { port: '30303-30315', comment: '{{ host_el_type }} discovery stable', protocol: 'udp'                     }
+    - { port: '6060-6064',   comment: 'EL Exporter',                         ipset: 'metrics.hq', iifname: 'wg0' }
+    - { port: '9400-9404',   comment: 'Go-Ethereum Exporter',                ipset: 'metrics.hq', iifname: 'wg0' }
```

What's the `stable` part for tho?
```suggestion
    - { port: '30303-30315', comment: '{{ host_el_type }} LibP2P',    protocol: 'tcp'                     }
    - { port: '30303-30315', comment: '{{ host_el_type }} Discovery', protocol: 'udp'                     }
    - { port: '6060-6064',   comment: '{{ host_el_type }} Metrics',                         ipset: 'metrics.hq', iifname: 'wg0' }
    - { port: '9400-9404',   comment: '{{ host_el_type }} Exporter',                ipset: 'metrics.hq', iifname: 'wg0' }
```